app/suggestion/views.py

```diff
@@ -2,7 +2,13 @@
 from flask.ext.login import login_required
 from . import suggestion
 
+
 @suggestion.route('/')
 @login_required
 def index():
     return render_template('suggestion/index.html')
+
+
+@suggestion.route('/suggest-new')
```

Comment:
since reviewing the new suggestions will be within the same blueprint, I think just "new" will be ambiguous.